internal/commands/untrust_builder.go

```diff
@@ -28,7 +34,8 @@ func UntrustBuilder(logger logging.Logger, cfg config.Config) *cobra.Command {
 			}
 
 			if len(existingBuilders) == len(cfg.TrustedBuilders) {
-				logger.Infof("Builder %s wasn't trusted", style.Symbol(builderName))
+				// Attempted to untrust a suggested builder
+				logger.Warnf("Builder %s currently can't be untrusted", style.Symbol(builderName))
```

Comment:
Can we be more explicit about why? Something like `builders that are listed in suggested-builders are trusted by default. Currently pack doesn't support making these builders untrusted`